examples/README.md

```diff
@@ -2,9 +2,16 @@
 
 # Examples
 
-This directory contains fully functional code examples that you can use to learn more about Objax
+This directory contains fully functional code examples that you can use to learn more about Objax.
 
-See:
-* [Image Classification](classify/img/README.md)
+Examples from classic machine learning:
+* [Image Classification](image_classification/README.md)
+* [Text Generation](text_generation/README.md)
+
+Examples from recent research:
 * [Model-Agnostic Meta-Learning](optimization/README.md)
-* [Recurrent Neural Networks (RNN)](rnn/README.md)
+* [FixMatch](fixmatch/README.md)
```

Comment:
** Comments about fixmatch move:

1) it seems like it moved to `examples/fixmatch/img/`. However right now we only have Fixmatch for images and it's unlikely that we will have fixmatch for other domains in any near future. This this Fixmatch should be moved to `examples/fixmatch`

2) You need to update all imports in fixmatch code.
It seems like right now fixmatch files are moved to new location, however they still use old imports like `from examples.classify.semi_supervised.img... import ...`

3) Also fixmatch path need to be updated in the `example/fixmatch/README.md`

Reply:
Done.